scripts/generateDc2InstCat.py

```diff
@@ -26,6 +29,12 @@ def get_maskedMagNorm(self):
             return raw_norm
         return np.where(raw_norm<self.min_mag, self.min_mag, raw_norm)
 
+    def column_by_name(self, colname):
+        if (not colname.startswith('properMotion')
```

Comment:
@danielsf Should the radial velocity also be zeroed here, as you suggested in #29 ?